docs/docs/documentation/configuration-file.md

```diff
@@ -188,7 +188,7 @@ performance = "High"
 #
 # Example 1 using fish shell from bin path:
 #
-#   shell = { program = "/bin/fish", args = ["--login"] }
+#  shell = { program = "/bin/fish", args = ["--login"] }
```

Comment:
maybe this one needs to delete just one more space?

Reply:
good catch. I should also update the `rio-config/src/defaults.rs` file accordingly, right?

Reply:
done ✅